kuronometer-core/src/test/scala/com/github/pedrovgs/kuronometer/generators/ConfigGenerators.scala

```diff
@@ -0,0 +1,17 @@
+package com.github.pedrovgs.kuronometer.generators
+
+import com.github.pedrovgs.kuronometer.free.domain.{Config, Platform}
+import org.scalacheck.{Arbitrary, Gen}
+
+object ConfigGenerators {
+
+  implicit lazy val arbConfig: Arbitrary[Config] = Arbitrary(config())
```

Comment:
The ``arbString`` implementation from ``ScalaCheck`` is declared as lazy. But you are totally right, declaring it as lazy it's not needed. Fixed!